java/code/webapp/WEB-INF/pages/systems/sdc/overview.jsp

```diff
@@ -146,6 +146,7 @@
                 </c:when>
                 <c:otherwise>
                   <c:out value="${system.runningKernel}" />
+                  <a href="/rhn/manager/contentmanagement/filters?openFilterId=-1&openTemplate=LivePatchingSystem&systemId=${system.id}&systemName=${system.name}&kernelId=0000&kernelName=${system.runningKernel}">(<bean:message key="sdc.details.overview.create_clm_filter"/>)</a>
```

Comment:
The `kernelId=0000` bit is the part I couldn't figure out, how can I figure out what the relevant kernel id is from the system details?

Reply:
This was also the issue with the initial implementation of the form. The problem is that kernel string is reported by salt directly from the system and it does not match a package name.

My suggestion is to just leave the kernel input as-is. It should show the latest kernel selected. This is how the form normally works anyway. Only thing the link has to do is to preselect a system, then the kernel input will automatically select the latest available. Most of the time, systems run the latest installed kernel anyway.

Reply:
That sounds like a good compromise, thanks for the suggestion. I've updated the relevant bits.

Reply:
So IIUC we do not provide any `kernel` hints for the target template, we just point directly to the live patching template for the system in the context. Then my question is: is it fine to have the link right next to the `kernel` entry, or should we put it generically next to the SUSE Manager System ID instead? Or next to the Live Patching `Installed Products:` entry instead? IIRC the Live Patching is an installed product, isn't it @cbbayburt ? I am not suggesting to move it already, I am more thinking out loud looking for reasoning, you know :grinning:

Reply:
> IIRC the Live Patching is an installed product, isn't it @cbbayburt?

@ncounter you're right, but when you want to use CLM for live patching, you wouldn't install Live Patching directly from vendor channels. Therefore, you won't see it here until after you set up the CLM and assign the system to it.

OTOH, putting it next to the system ID seems too general to me. Only thing that's relevant to live patching is the kernel field on this page. Also keep in mind that when a live patch is actually installed on a system, the current live patch version shows up right below the kernel value here in the same field.

Reply:
Thanks @cbbayburt for the clarification. I fully agree with the current solution now 👍